src/js/components/Anchor/StyledAnchor.js

```diff
@@ -13,23 +13,26 @@ export const StyledAnchor = styled.a`
   font-size: inherit;
   line-height: inherit;
   color: ${props => normalizeColor(props.colorProp || props.theme.anchor.color, props.theme)};
-  ${props => props.theme.anchor.fontWeight
-    && `font-weight: ${props.theme.anchor.fontWeight};`}
+  ${props => props.theme.anchor.fontWeight && `font-weight: ${props.theme.anchor.fontWeight};`}
   text-decoration: ${props => (props.hasIcon ? 'none' : props.theme.anchor.textDecoration)};
   cursor: pointer;
   outline: none;
   ${genericStyles}
 
-  ${props => !props.disabled && props.theme.anchor.hover && css`
+  ${props =>
+    !props.disabled &&
+    props.theme.anchor.hover &&
+    css`
     &:hover {
-      ${props.theme.anchor.hover.textDecoration
-        && `text-decoration: ${props.theme.anchor.hover.textDecoration};`}
-      ${props.theme.anchor.hover.fontWeight
-        && `font-weight: ${props.theme.anchor.hover.fontWeight};`}
+      ${props.theme.anchor.hover.textDecoration && `text-decoration: ${props.theme.anchor.hover.textDecoration};`}
```

Comment:
This line seems too long. I have been slowly going through the files as I touch them to make sure they fit within 80 columns.

Reply:
yeah I agree that 80 columns is best. let me try it again, but there were some minor changes to the snapshots since the template literals were breaking in weird spots.